isamples_metadata/SESARTransformer.py

```diff
@@ -309,7 +309,7 @@ def has_material_categories(self) -> typing.List[str]:
             sesar_model = MetadataModelLoader.get_sesar_material_model()
             # load the model predictor
             smp = SESARMaterialPredictor(sesar_model)
-            categories = [smp.predict_material_type(self.source_record).value]
+            categories = [prediction.value for prediction in smp.predict_material_type(self.source_record)]
```

Comment:
Do we expect these to actually return multiple useful values for SESAR?  Or do we still expect one?

Reply:
Thinking again, I think there might be an issue here. The test records currently contain the top3 high confidence labels, and the confidence of each of it (when we are doing machine prediction, if it is just rule-based then there would be only 1 value in the list). But the `has_material_category` only returns a list of labels. I think we might need an additional field to store confidence scores as well.

Reply:
Yeah, for now we are ignoring the confidence values in the test because we don't have them stored anywhere.

Reply:
(And confirmed that the tests pass with the multiple values)